docs/data/sdks/typescript-react-native/index.md

```diff
@@ -253,14 +253,35 @@ identify(identifyObj);
 
 --8<-- "includes/groups-intro-paragraph.md"
 
-```ts
-import { setGroup } from '@amplitude/analytics-react-native';
+!!! example
+    If Joe is in 'orgId' '15', then the `groupName` would be '15'.
+
+    ```ts
+    import { setGroup } from '@amplitude/analytics-react-native';
+
+    // set group with single group name
+    setGroup('orgId', '15');
+    ```
+
+    If Joe is in 'sport' 'tennis' and 'soccer', then the `groupName` would be '["tennis", "soccer"]'.
+
+    ```ts
+    import { setGroup } from '@amplitude/analytics-react-native';
 
-// set group with single group name
-setGroup('orgId', '15');
+    // set group with multiple group names
+    setGroup('sport', ['soccer', 'tennis']);
+    ```
+
+--8<-- "includes/event-level-groups-intro.md"
+
+```ts
+import { track } from '@amplitude/analytics-react-native';
 
-// set group with multiple group names
-setGroup('sport', ['soccer', 'tennis']);
+track({
+  event_type: 'event type',
+  groups: { 'orgId': '15' },
```

Comment:
Nits, Could we change the order by event related info first, then go to group info?  There have other places that also have the same nits. 
track({
  event_type: 'event type',
  event_properties: { eventPropertyKey: 'event property value' }
  groups: { 'orgId': '15' },
})

Reply:
Fixed.